extras/ams_wrapper/src/config.py

```diff
@@ -42,4 +42,13 @@
         },
     "config_path": "/opt/ams_models/vehicle_attributes_model.json"
     },
+    {"name": "emotionsRecognition", 
+    "class": VehicleAttributes, 
```

Comment:
I think this class name is a little misleading 😄

Reply:
Done.